packages/web-comment/src/routes/appeal-search-no-results/format-no-results-page.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @param {string} searchQuery
+ * @returns {string}
+ */
+exports.isPostcodeOrReferenceNumber = (searchQuery) => {
```

Comment:
before either of these regexes went in I'd want a lot of unit tests for them
however rather than managing a complex regex, aren't we just passing through a query string telling us this on redirect?

```
res.redirect(`appeal-search-no-results?search=
```

Reply:
I've noticed this is also used in the enter-postcode controller, I don't think it should be

Reply:
I removed the regex and added the search type as a param in the redirect. This wasnt already done if thats what you meant

Reply:
ah yes sorry was being dumb, but yes I think this is preferable

Reply:
yea it definitely seems like a smoother approach.

Think changing or amending the regex in the other controller should probably be a separate ticket?

Reply:
yes, will attempt to think of a simpler approach